lib/address_space/add-enumeration-type/install.js

```diff
@@ -27,7 +24,8 @@ function install(AddressSpace) {
      * @param options.enumeration[].value       {Number}
      * @param options.enumeration[].description {String|LocalizedText|null}
      */
-  AddressSpace.prototype.addEnumerationType = function (options) {
+    /* eslint-disable max-len */
```

Comment:
Why change indent here ? AddressSpace.prototype.addEnumerationType  is inside the install function and shall be indented?